RSSParser/RSSParser.m

```diff
@@ -109,6 +109,15 @@ - (void)parser:(NSXMLParser *)parser didEndElement:(NSString *)elementName names
             }
         } else if ([elementName isEqualToString:@"content:encoded"] || [elementName isEqualToString:@"content"]) {
             [self.currentParsedItem setContent:self.currentParsedText];
+
+            NSString *string = self.currentParsedText;
+            if ([string hasPrefix:@"<p><!--"] || [string hasPrefix:@"<!--"]) {
```

Comment:
Is there any chance that the string might also be prefixed with white space?

Reply:
Hmmm... I just tested it on Diocesan's Wordpress. Added a couple spaces _before_ the comment string and they do indeed get sent in the parsed `content:encoded` tag. So I guess I should remove that check and just use the Regex?

Reply:
That, or trim the whitespace